### ISSUE TITLE
Validation and training sets should be different

### ISSUE THREAD

**Issue by Esterci** [ybwerneck/Pinn-Torch#4]

**Comment by ybwerneck** [comment]
I don't remember however if the loss is used in early stopping or the scheduler, if so it should not  (its cross information) and we should change to use the validation  (maybe set up the scheduler already informing the marker to track)?
